docs/appendices/release-notes/unreleased.rst

```diff
@@ -252,3 +252,24 @@ Fixes
     UNION
     SELECT i, avg(k), j FROM t3 GROUP BY 1, 3
     LIMIT 10
+
+- Added a workaround for an issue that caused an exception to be thrown when
```

Comment:
Why is this a workaround from the user perspective? If so, we would have to explain why we think this is a workaround, I think.
```suggestion
- Fixed an issue that caused an exception to be thrown when
```